### PR TITLE
ci: Update the release workflow (no-changelog)

### PR DESCRIPTION
**Pull request by StarfallProjects**
Add a step to the release action to trigger release notes.

Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: .github/workflows/release-publish.yml [hunk 1]
```diff
@@ -41,6 +41,9 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
           pnpm publish -r --publish-branch ${{github.event.pull_request.base.ref}} --access public --tag rc
+          
+      - name: Trigger a release note
+        run: curl --location --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' --data '{"version":${{env.RELEASE}}}'
```
Comment (StarfallProjects) [inline .github/workflows/release-publish.yml:46]:
```suggestion
        run: curl --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' --data '{"version":${{env.RELEASE}}}'
```

File: .github/workflows/release-publish.yml [hunk 2]
```diff
@@ -41,6 +41,9 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
           pnpm publish -r --publish-branch ${{github.event.pull_request.base.ref}} --access public --tag rc
+          
+      - name: Trigger a release note
+        run: curl --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' --data '{"version":${{env.RELEASE}}}'
```
Comment (StarfallProjects) [inline .github/workflows/release-publish.yml:46]:
```suggestion
        run: curl --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' --data '{"version":"${{env.RELEASE}}"}'
```

File: .github/workflows/release-publish.yml [hunk 3]
```diff
@@ -41,6 +41,9 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
           pnpm publish -r --publish-branch ${{github.event.pull_request.base.ref}} --access public --tag rc
+          
+      - name: Trigger a release note
+        run: curl --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' --data '{"version":"${{env.RELEASE}}"}'
```
Comment (StarfallProjects) [inline .github/workflows/release-publish.yml:46]:
```suggestion
        run: curl --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' -- header 'Authorization:  Basic [TODO]' --data '{"version":"${{env.RELEASE}}"}'
```

File: .github/workflows/release-publish.yml [hunk 4]
```diff
@@ -41,6 +41,9 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
           pnpm publish -r --publish-branch ${{github.event.pull_request.base.ref}} --access public --tag rc
+          
+      - name: Trigger a release note
+        run: curl --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' -- header 'Authorization:  Basic [TODO]' --data '{"version":"${{env.RELEASE}}"}'
```
Comment (StarfallProjects) [inline .github/workflows/release-publish.yml:46]:
```suggestion
        run: curl --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' -- header 'Authorization:Basic [TODO]' --data '{"version":"${{env.RELEASE}}"}'
```

File: .github/workflows/release-publish.yml [hunk 5]
```diff
@@ -41,6 +41,9 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
           pnpm publish -r --publish-branch ${{github.event.pull_request.base.ref}} --access public --tag rc
+          
+      - name: Trigger a release note
+        run: curl --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' -- header 'Authorization:Basic [TODO]' --data '{"version":"${{env.RELEASE}}"}'
```
Comment (StarfallProjects) [inline .github/workflows/release-publish.yml:46]:
```suggestion
        run: curl -u docsWorkflows:${{ secrets.N8N_WEBHOOK_DOCS_PASSWORD }} --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' --data '{"version":"${{env.RELEASE}}"}'
```

**Review by netroy — DISMISSED** [review]

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.227.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.227.0)

**Pull request merged**